docs/source/usage/plugins/openPMD.rst

```diff
@@ -0,0 +1,118 @@
+.. _usage-plugins-openPMD:
+
+openPMD
+-----
+
+Stores simulation data such as fields and particles using the `openPMD API <https://openpmd-api.readthedocs.io>`_.
+
+External Dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+The plugin is available as soon as the :ref:`openPMD API <install-dependencies>` is compiled in.
```

Comment:
Since the current release does not yet include JSON configurability, I've specified `- 0.12.0+ (yet to be released)` in `INSTALL.rst`. I have not yet update `CMakeLists.txt` since the current dev of the openPMD API still goes with the old version.